e2e/jenkins/crw-ci/k8s/pr-check/Jenkinsfile

```diff
@@ -0,0 +1,229 @@
+#!groovy
+
+pipeline {
+    agent { label 'codenvy-slave9' }
+
+    options {
+        timestamps()
+        buildDiscarder(logRotator(artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '', daysToKeepStr: '60', numToKeepStr: '100'))
+    }
+
+    // triggers { cron("@midnight") }
+
+    environment {
+        PATH = "/tmp:/qa/tools/opt/apache-maven-3.5.4/bin:$PATH"
+        JENKINS_BUILD = "true"
+
+        DEVFILE_URL = "${WORKSPACE}/e2e/files/happy-path/happy-path-workspace.yaml"
+        SUCCESS_THRESHOLD = 5
+
+        CHE_IMAGE_REPO = "maxura/che-server"
+        CHE_IMAGE_TAG = "${ghprbPullId}"
+    }
+
+    stages {
+        stage("Prepare codebase") {
+            failFast true
+
+            parallel {
+                stage("Cleanup") {
```

Comment:
Wouldn't it be better to do cleanup in post actions? Using
```
cleanWs notFailBuild: true
```

Reply:
There could be builds aborted before, so we should insure `codenvy-slave9` is ready for usage.

Reply:
And `cleanWs notFailBuild: true` is suitable here as well. Thanks for advice.

Reply:
UPDATE: `cleanWs notFailBuild: true` IS NOT suitable here. I have mislead that command propose.